### PR TITLE
Waiters tab for Administrator panel

### PR DESCRIPTION
**Pull request by MaxSukhovetskyi**
NOTICE:
1.  Changed typo in order status from "Asigned waiter" to "Assigned waiter"
2.  Changed db init script! (previous user might not be working, also added a lot more users)
3.  Changed db init script to create orders for every status and restaurant

### CODE REVIEW

File: easyrest/views/auth_controler.py [hunk 1]
```diff
@@ -33,8 +33,6 @@ def login_post(request):
 
     if user is None or not pbkdf2_sha256.verify(password, user.password):
         raise HTTPForbidden("Email or password is invalid")
-    elif not user.is_active:
```
Comment (RazSKg) [inline easyrest/views/auth_controler.py]: Why?

File: easyrest/views/order_controller.py [hunk 2]
```diff
@@ -45,16 +45,25 @@ def get_order(request, order_id, filter_list=False):
 
 
 @view_config(route_name='get_orders', renderer='json', request_method='GET')
-@restrict_access(user_types=["Client"])
+@restrict_access(user_types=["Administrator"])
 def get_orders(request):
-    """Controller for get list of user orders without items
+    """Controller for get list of user orders without items for Administrator
```
Comment (RazSKg) [inline easyrest/views/order_controller.py]: really users?

File: frontend/src/components/AdministratopPanel/AdministratorPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  handleSnackbar = (msg, type) => {
+    this.setState({
+      SnackbarMsg: msg,
+      isSnackbarOpen: true,
+      SnackbarType: type
+    });
+  };
+
+  changeStatus = (orderIndex, newStatus) => {
+    console.log(this.state.orders[orderIndex], orderIndex);
```
Comment (RazSKg) [inline frontend/src/components/AdministratopPanel/AdministratorPanel.jsx]: do not commit debugging code!

File: frontend/src/components/AdministratopPanel/OrderListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    flex: 1
+  },
+  details: {
+    flex: 9999,
+    minWidth: "25em",
+    display: "flex"
+  }
+});
+
+function handlerChangeDate(date) {
```
Comment (RazSKg) [inline frontend/src/components/AdministratopPanel/OrderListItem.jsx:29]: You can use date fns in service.

**Review by RazSKg — APPROVED** [review]

File: easyrest/views/auth_controler.py (hunk 1 above)
Comment (MaxSukhovetskyi) [reply easyrest/views/auth_controler.py]: Git bug

**Pull request merged**